### ISSUE TITLE
Validation of file

### ISSUE THREAD

**Issue by gcelano** [PerseusDL/canonical-latinLit#484]
I tried to validate the file at  https://github.com/PerseusDL/canonical-latinLit/blob/master/data/phi0448/phi001/phi0448.phi001.perseus-lat2.xml against http://epidoc.stoa.org/schema/8.19/tei-epidoc.rng (this url schema is given in the XML file), but I get an error message: can you validate it successfully?

**Comment by lcerrato** [comment]
Hi @gcelano 
You can check:
https://app.travis-ci.com/github/PerseusDL/canonical-latinLit/builds/251339319
to see what files are passing/failing. This appears to be passing.

it also appears in Scaife so that would not happen if it were failing:
https://scaife.perseus.org/reader/urn:cts:latinLit:phi0448.phi001.perseus-lat2:1.1.1-1.1.5/

**Comment by lcerrato** [comment]
Do you have info on the error message(s)?

**Comment by gcelano** [comment]
Yes, it works. There was a problem with the namespace in my copy of the original XML file. Thanks!
